test/libsolidity/syntaxTests/tight_packing_literals_fine.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+    function f() pure public returns (bytes32) {
+        return keccak256(uint8(1));
+    }
+    function g() pure public returns (bytes) {
+        return abi.encode(1);
```

Comment:
This isn't tight packing? Should instead have the same 5 methods as `tight_packing_literals.sol`

Reply:
The idea was that these are the "good" cases that are variations of `tight_pcaking_literals.sol`. Perhaps the naming of the file is bad.